app/utils/homepageManagement.js

```diff
@@ -33,6 +33,20 @@ export async function sendEmail(formData, pageInfo) {
   return true;
 }
 
+/* istanbul ignore next */
+export function scrollToSection() {
```

Comment:
Move to another utility file with a descriptive name. The function is used not just on home page.